ipaddr/ipaddr.go

```diff
@@ -4,8 +4,23 @@ import (
 	"fmt"
 	"net"
 	"reflect"
+	"strings"
 )
 
+// AddressIsIpv6 detected whether the argument is an IPv6 Address
+func AddressIsIpv6(address string) bool {
+	ip := net.ParseIP(address)
+	return ip != nil && strings.Contains(address, ":")
+}
+
+// FormatAddressPort format a tuple address/port that works with IPv6
+func FormatAddressPort(address string, port int) string {
```

Comment:
I feel like this should just use the stdlib function `net.JoinHostPort` instead: https://golang.org/pkg/net/#JoinHostPort

Reply:
@r.boyer huh... nice... Did not found this one. Would you me to keep this function and to use https://golang.org/pkg/net/#JoinHostPort or to rewrite everything?

Reply:
@rboyer DONE, I kept `ipaddr.FormatAddressPort(string, int)` since `JoinHostPort(string, string)` is string only